src/test/java/org/opensearch/knn/index/codec/KNNCodecTestCase.java

```diff
@@ -317,6 +320,7 @@ public void testKnnVectorIndex(
         IndexWriterConfig iwc = newIndexWriterConfig();
         iwc.setMergeScheduler(new SerialMergeScheduler());
         iwc.setCodec(codec);
+        iwc.setMergePolicy(NoMergePolicy.INSTANCE);
```

Comment:
Would be nice to have a comment explaining why we need this policy.
Shouldn't we modify the test itself so that it won't fail even without specifying this policy?

Reply:
More specific test with less chance of missing fault is better. In that sense, using NoMergePolicy is better I guess.

Reply:
Sounds good. I updated and added a comment.